.github/workflows/phpunit.yml

```diff
@@ -17,14 +17,18 @@ jobs:
     - name: Validate composer.json and composer.lock
       run: composer validate
 
-    - name: Cache Composer packages
```

Comment:
you should just use https://github.com/ramsey/composer-install - this would simplify your action and you don't need to work with this ugly github/action cache key things

Reply:
That point is correct, but adding an informal dependency on action requires trust in the action developer. Leave the decision to the administrator of this package.